web_src/css/repo.css

```diff
@@ -618,6 +629,11 @@ td .commit-summary {
   border: 1px solid var(--color-light-border);
 }
 
+.repository.view.issue .comment-list .timeline-item.with-labels-list .ui.label {
+  margin: 0 2px;
```

Comment:
.25rem at 14px font size is 3.5px, so half of it should be 1.75px, not 2px. I think we should add a `--labels-gap: 4px` variable and divide by two here.

Reply:
Or if we can make it `a.ui.label`, the selector could be `.label + .label` which would be more clean.

Reply:
Or maybe name it `--gap-labels` to be more consistent with existing vars.

Reply:
There is no `a.ui.label` because we are using `<a><span class="ui label"></span><a>`

I made new changes in latest commits.

Reply:
I have the same feeling but it is more difficult. Need much more changes including backend render function changes.